### PR TITLE
fix: use config value in configmap

### LINKED ISSUE #13: Configuration is not working
Hi team,

I've seen that we have a `config` value, which makes nothing. I see that the configMap is not a template.

### PR DESCRIPTION
**Pull request by runarheggset**
Closes https://github.com/softonic/go-ratelimit-chart/issues/13.

### CODE REVIEW

**Comment by santinoncs** [comment]
Thanks a lot for your contribution.

File: values.yaml [hunk 1]
```diff
@@ -65,7 +65,12 @@ extraEnvs: []
 logLevel: debug
 
 config: |-
-  # Start with empty config
+  domain: edge_proxy_per_ip
```
Comment (santinoncs) [inline values.yaml]: why you need this config here?
The purpose of initconfig is to start the daemon with a default config.
So i see a good idea to add the config's value in the values.yaml but i don't understand why you set the value "edge_proxy_per_ip" here

Comment (runarheggset) [reply values.yaml]: It's just an example config, so it's not really needed. Changed it back to the previous example config.

Comment (santinoncs) [reply values.yaml]: ok no worries
is just out of curiosity
But we can keep this one, if it is working

**Review by santinoncs — APPROVED** [review]

**Pull request merged**